src/clients/dotnet/ci.zig

```diff
@@ -9,6 +9,7 @@ const Shell = @import("../../shell.zig");
 const TmpTigerBeetle = @import("../../testing/tmp_tigerbeetle.zig");
 
 pub fn tests(shell: *Shell, gpa: std.mem.Allocator) !void {
+    assert(shell.file_exists("TigerBeetle.sln"));
```

Comment:
Fun fact: I've added these asserts as a drive by change, because why not, and they've already caught a bug in this PR!